src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyExecutionManagerWithDataCollection.cs

```diff
@@ -22,6 +22,7 @@ internal class ProxyExecutionManagerWithDataCollection : ProxyExecutionManager
         private IDictionary<string, string> dataCollectionEnvironmentVariables;
         private int dataCollectionPort;
         private IRequestData requestData;
+        private TestRunCriteria testRunCriteria;
```

Comment:
remove testRunCriteria entirely. It will go directly to ProxyDataCollectionManager from TestEngine

Reply:
Done